migrations/20260703235904_rekey_tokenized_asset_aggregate_id.sql

```
@@ -32,18 +32,68 @@ WHERE aggregate_type = 'TokenizedAsset'
-UPDATE snapshots
+CREATE TEMP TABLE reseq AS
+SELECT rowid AS event_rowid,
+       ROW_NUMBER() OVER (
+           PARTITION BY aggregate_type, aggregate_id ORDER BY sequence
+       ) AS new_sequence
+FROM events
+WHERE aggregate_type IN ('TokenizedAsset', 'Underlying');
+
+UPDATE events
+SET sequence = (
+    SELECT new_sequence FROM reseq WHERE event_rowid = events.rowid
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)